examples/ros_bag_recorder_adapter/adapter/adapter.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/python
+"""
+This file contains the definition of the Adapter itself. The adapter is what allows us to start up the bag recorder
+"""
+
+import logging
+import threading
+import time
+from typing import Tuple
+
+from baghandler import BagHandler
+from utils import *
+
+# logger.basicConfig(level=get_log_level())
+logger = logging.getLogger(__name__)
+
+def save_msg_to_bag(data, callback_params: Tuple[BagHandler, str]):
```

Comment:
if this is just a function on the adapter class then you don't need to pass in the BagHandler